pytest-embedded-serial-esp/pytest_embedded_serial_esp/serial.py

```diff
@@ -133,7 +133,7 @@ def _post_init(self):
 
         super()._post_init()
 
-    def use_esptool(hard_reset_after: bool = True, no_stub: bool = False):
```

Comment:
breaking change...

we should deprecate this param if it's no longer useful.

Reply:
btw, is hard_reset_after still useful?

Reply:
We can move it into `esptool.main`.